index.mjs

```diff
@@ -0,0 +1,121 @@
+import jestPlugin from "eslint-plugin-jest";
+import nPlugin from "eslint-plugin-n";
+import importPlugin from "eslint-plugin-import";
+import jsonPlugin from "eslint-plugin-json";
+import eslint from "@eslint/js";
+import perfectionistPlugin from "eslint-plugin-perfectionist";
+import prettierPluginRecommended from "eslint-plugin-prettier/recommended";
+import tseslint from "typescript-eslint";
+import sonarjsPlugin from "eslint-plugin-sonarjs";
+import unicornPlugin from "eslint-plugin-unicorn";
+import tsParser from "@typescript-eslint/parser";
+
+export default tseslint.config(
+  {
+    files: ["**/*.ts", "**/*.tsx"],
```

Comment:
This is what I mention in the PR. This is the replacement of the `-- ext` option. Maybe we should include here `*.js` and `*.jsx`?

Reply:
I'll try on a JS project to see what happens. It might just work fine. In this repo I had to add some tweaks for the default config so it could pick them

Reply:
We have a separate eslint config repo for `*.js`, right?
Although I guess the point is that in TS projects we may have `.js` files,... So I guess we could try adding them here and if there are problems, adding a separate config as Esau mentions

Reply:
> We have a separate eslint config repo for *.js, right?

We do, but that project is not used at all as we don't have any JS only project 

I'll play with things and see if it works!

Reply:
> We do, but that project is not used at all as we don't have any JS only project

`web-schumacher-app` still relies on it for `.js`!

Reply:
I haven't really added all of our rules from `open-turo/eslint-config`, but now we have a section in this config to configure rules just for JS files

https://github.com/open-turo/eslint-config-typescript/pull/373/files#diff-f61fbd23dd2c16becb15b6d32c5147d13587b08a6dc7280a12d2170c4db7faeaR31

```js
const javascriptConfig = (ecmaVersion = "latest") =>
  // ...
```

This would allow us to deprecated the other one eventually in favor of this one